src/main/resources/db/migration/V215__fagsak_type.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE FAGSAK ADD COLUMN type VARCHAR(50) DEFAULT 'NORMAL' NOT NULL;
+ALTER TABLE FAGSAK ADD COLUMN fk_institusjon_id BIGINT;
+ALTER TABLE FAGSAK
+    ADD FOREIGN KEY (fk_institusjon_id) REFERENCES INSTITUSJON (ID);
+
+CREATE UNIQUE INDEX uidx_fagsak_type_aktoer_institusjon_ikke_arkivert ON fagsak(type, fk_aktoer_id, fk_institusjon_id)
+    WHERE arkivert = false;
```

Comment:
Fant ut at det gikk å ha flere fagsaker med samme type og aktør når institusjon er null, så før vi deployer til miljø bør vi endre til noe sånt som dette:
```suggestion
CREATE UNIQUE INDEX uidx_fagsak_type_aktoer_institusjon_ikke_arkivert ON fagsak(type, fk_aktoer_id, fk_institusjon_id)
    WHERE  fagsak.fk_institusjon_id IS NOT NULL
    AND arkivert = false;

CREATE UNIQUE INDEX uidx_fagsak_type_aktoer_ikke_arkivert ON fagsak(type, fk_aktoer_id)
    WHERE  fagsak.fk_institusjon_id IS NULL
    AND arkivert = false;
```